x-pack/legacy/plugins/siem/public/components/page/add_to_kql/index.tsx

```diff
@@ -6,56 +6,46 @@
 
 import { EuiIcon, EuiPanel, EuiToolTip } from '@elastic/eui';
 import { Filter } from '@kbn/es-query';
-import { getOr } from 'lodash/fp';
 import React from 'react';
-import { connect } from 'react-redux';
 import styled from 'styled-components';
-import { StaticIndexPattern } from 'ui/index_patterns';
-import { Query } from 'src/plugins/data/common';
 
 import { WithHoverActions } from '../../with_hover_actions';
-import { InputsModelId } from '../../../store/inputs/constants';
 import { siemFilterManager } from '../../search_bar';
 
 import * as i18n from './translations';
-import { filterQuerySelector } from '../../search_bar/selectors';
-import { State } from '../../../store';
-import { InputsRange } from '../../../store/inputs/model';
+import { InputsModelId } from '../../../store/inputs/constants';
 
 export * from './helpers';
 
-interface AddToKqlRedux {
-  query: Query;
-}
-
 interface OwnProps {
   id: InputsModelId;
```

Comment:
I think this can be optional or removed, I think AddKql is only used on `global` for now. So even if you add the `timeline`, it won't work. I think that I will get rid of it and maybe we can change the name of the component to `AddKqlToGlobal`. What do you think?

Reply:
maybe even better naming `AddFilterToGlobalSearchBar`

Reply:
I had left this & `InputsModelId` in so you would have it when plumbing through the SearchBar in timeline -- happy to remove though. Would prefer having a single `AddToKql` component with different props for where it's sending it vs a separate `<AddToKQLGlobal>` & `<AddtoKQLTimeline>`.

Reply:
we are not using `<AddToKql>` at all in the timeline, so when we need it we can always refactor the `<AddFilterToGlobalSearchBar>` to `<AddFilterToSearchBar>`. I think it will be easier for other engineers to exactly know what's going on here.

Reply:
Sounds good -- refactored to `AddFilterToGlobalSearchBar` 🙂